tools/lint.py

```diff
@@ -82,7 +82,7 @@ def run_cython_lint(files):
     if not files:
         return 0, ""
     res = subprocess.run(
-        ['cython-lint'] + files,
+        ['cython-lint', '--no-pycodestyle'] + files,
```

Comment:
Is it normal that the linter was modified? Doesn't seem like something that should be in this PR

Reply:
This won't be needed once #17917 is merged

Reply:
The above is merged, so this is no longer a diff.